cfgov/unprocessed/css/enhancements/tables.less

```diff
@@ -1,66 +1,9 @@
-/* topdoc
-  name: Tables
-  family: cf-tables
-  notes:
-    - Adjustments for cf-tables to allow them to be full-width
-  tags:
-    - cfgov-cf-tables
-*/
-
 .respond-to-max( @bp-sm-max, {
   table {
     width: 100%;
   }
 } );
 
-/* topdoc
-  name: Table Organism
-  family: cfgov-cf-enhancements
-  notes:
-    - Extends the above stacked tables for sm devices
-    - Hides the thead
-    - Styles first td as if it were the thead
-    - Allows any other td to be styled as a heading level 4
-    - Adds data-label attr for small td labels above td content
-    - Used in older blog posts to make them responsive.
-      Do not remove unless removed from blog posts.
```

Comment:
This seems like it may be important to retain.

Reply:
The description in the notes is actually incorrect. It doesn't do these things it says it does. I actually specifically looked for places on our site where we have $$('.o-table tbody th') (which looked a bit odd to me because it includes `th`s outside the `thead`, and found (at least): https://www.consumerfinance.gov/about-us/contact-us/. However, I never found this in a blog post.

It's actually not against any validation rules to but a th in the tbody, so this rule/matching markup seemed fine to keep.

Reply:
Oh, that comment is actually talking about adding the 100% to tables that don't have an `o-table` class. Seems like we should add this to cfpb-tables instead!